cmake/Platform/Linux/PackagingPostBuild_linux.cmake

```diff
@@ -8,6 +8,7 @@
 
 file(REAL_PATH "${CPACK_SOURCE_DIR}/.." LY_ROOT_FOLDER)
 include(${LY_ROOT_FOLDER}/cmake/Platform/Common/PackagingPostBuild_common.cmake)
+include(${LY_ROOT_FOLDER}/cmake/Platform/Common/PackagingCodeSign.cmake)
```

Comment:
here wouild be where we include the "current platform" instead
note that here we are executing in cpack environment, so we dont have access to `LY_PLATFORM_NAME` (would have to be passed to cpack)

Reply:
Sure, will do